doc/src/conf.py

```diff
@@ -84,24 +84,24 @@
-    "modules/physics/mechanics/autolev_parser": "../explanation/modules/physics/mechanics/autolev_parser.html",
-    "modules/physics/mechanics/examples": "../tutorials/physics/mechanics/index.html",
-    "modules/physics/mechanics/joints": "../explanation/modules/physics/mechanics/joints.html",
-    "modules/physics/mechanics/kane": "../explanation/modules/physics/mechanics/kane.html",
-    "modules/physics/mechanics/lagrange": "../explanation/modules/physics/mechanics/lagrange.html",
-    "modules/physics/mechanics/masses": "../explanation/modules/physics/mechanics/masses.html",
-    "modules/physics/mechanics/reference": "../explanation/modules/physics/mechanics/reference.html",
-    "modules/physics/mechanics/symsystem": "../explanation/modules/physics/mechanics/symsystem.html",
-    "modules/physics/mechanics/linearize": "../explanation/modules/physics/mechanics/linearize.html",
-    "modules/physics/mechanics/sympy_mechanics_for_autolev_uses": "../explanation/modules/physics/mechanics/sympy_mechanics_for_autolev_uses.html",
-    "tutorials/physics/biomechanics/biomechanics": "../explanation/modules/physics/biomechanics/biomechanics.html",
+    "modules/physics/vector/index": "../../../explanation/modules/physics/vector/index.html",
+    "modules/physics/vector/vectors": "../../../explanation/modules/physics/vector/vectors/vectors.html",
+    "modules/physics/vector/kinematics": "../../../explanation/modules/physics/vector/kinematics/kinematics.html",
+    "modules/physics/vector/advanced": "../../../explanation/modules/physics/vector/advanced.html",
+    "modules/physics/vector/fields": "../../../explanation/modules/physics/vector/fields.html",
+    "modules/physics/mechanics/index": "../../../explanation/modules/physics/mechanics/index.html",
+    "modules/physics/mechanics/advanced": "../../../explanation/modules/physics/mechanics/advanced.html",
+    "modules/physics/mechanics/autolev_parser": "../../../explanation/modules/physics/mechanics/autolev_parser.html",
+    "modules/physics/mechanics/examples": "../../../tutorials/physics/mechanics/index.html",
+    "modules/physics/mechanics/joints": "../../../explanation/modules/physics/mechanics/joints.html",
+    "modules/physics/mechanics/kane": "../../../explanation/modules/physics/mechanics/kane.html",
+    "modules/physics/mechanics/lagrange": "../../../explanation/modules/physics/mechanics/lagrange.html",
+    "modules/physics/mechanics/masses": "../../../explanation/modules/physics/mechanics/masses.html",
+    "modules/physics/mechanics/reference": "../../../explanation/modules/physics/mechanics/reference.html",
+    "modules/physics/mechanics/symsystem": "../../../explanation/modules/physics/mechanics/symsystem.html",
+    "modules/physics/mechanics/linearize": "../../../explanation/modules/physics/mechanics/linearize.html",
+    "modules/physics/mechanics/sympy_mechanics_for_autolev_uses": "../../../explanation/modules/physics/mechanics/sympy_mechanics_for_autolev_uses.html",
```

Comment:
This one did not work for me.

https://output.circle-artifacts.com/output/job/f66c2bb4-4e27-44d6-b538-717ad45af252/artifacts/0/doc/_build/html/modules/physics/mechanics/sympy_mechanics_for_autolev_uses.html